emf/loadflow_tool/helper.py

```diff
@@ -309,4 +309,20 @@ def get_metadata_from_filename(file_name):
             file_metadata["Model.domain"],\
             file_metadata["Model.forEntity"] = entity_and_area_list
 
-    return file_metadata
\ No newline at end of file
+    return file_metadata
+
+
+def export_model(network: pypowsybl.network, CGM_meta):
```

Comment:
profiles should be a list of profile string ala ["EQ", "SSH"] on the function call. Default in function call should None, as listis are not good if deified in as default function call. Replace none with list of "SV,SSH,TP,EQ"